src/blk/BlockDevice.cc

```diff
@@ -47,6 +47,12 @@
 
 using std::string;
 
+std::ostream& operator<<(std::ostream& os, const FAMode buffered)
+{
+  os << (buffered == FAMode::BUFFERED ? " (buffered)" : " (direct)");
```

Comment:
nit, i'd suggest avoid embedding the formatting stuff into the `operator<<`, in other words,

```suggestion
  os << (buffered == FAMode::BUFFERED ? "(buffered)" : "(direct)");
```

Reply:
Make for more code a the actually printing, but makes sense.
Will fix